components/style/values/computed/text.rs

```diff
@@ -101,3 +101,69 @@ impl ToCss for TextDecorationLine {
         Ok(())
     }
 }
+
+macro_rules! define_text_align {
+    ($($name: ident($string: expr) => $discriminant: expr,)+) => {
+        define_css_keyword_enum! { TextAlign:
+            $(
+                $string => $name,
+            )+
+        }
+
+        impl TextAlign {
+            /// Convert computed TextAlign into u32
+            pub fn to_u32(self) -> u32 {
+                match self {
```

Comment:
This is super stupid btw (not your fault of course). I should just land the patch to `derive(Parse)` for keywords and use `self as u32`.

Reply:
I'll actually land it, gimme a sec :)

Reply:
#19578